package.json

```
@@ -42,5 +42,6 @@
     "stylelint": "14.2.0",
     "stylelint-config-standard": "23.0.0",
     "typescript": "4.4.3"
-  }
+  },
+  "version": "0.0.0"
```

Unrelated editor change committed by mistake?
```suggestion
  }
```